packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -676,6 +686,24 @@ const Combobox = forwardRef<
             </span>
           )}
         </label>
+        <div>
```

Comment:
I don't think you need the extra wrapping div, the other implementations do not have it.

```suggestion
```